docker-compose.yaml

```diff
@@ -0,0 +1,70 @@
+version: "3.8"
+services:
+  ceph-spdk:
```

Comment:
```suggestion
  # Fix ownership of Ceph conf directory
  # ceph-cluster         | ./vstart.sh: line 670: /etc/ceph/ceph.conf: Permission denied
  # By default, the volume directory is owned by user 'root'
  # ceph-cluster         | drwxr-xr-x 2 root root 4096 May  9 14:05 /etc/ceph
  # however ceph-cluster runs as user 'ceph'
  # ceph-cluster         | uid=167(ceph) gid=167(ceph) groups=167(ceph)
  # So this helps correct it
  change-vol-ownership:
    # We can use any image we want as long as we can chown
    image: alpine
    # Priviliged enough to chown ;)
    user: "root"
    volumes:
      # The volume to chown
      - ceph-conf:/etc/ceph
    command: chown -R 167:167 /etc/ceph
  ceph-spdk:
```

Hello @epuertat 🖖,

Thank you for the impressive demo.

Trying to reproduce:
```shell
➜  ceph-nvmeof git:(build-container) ✗ make up                               
docker-compose up --detach --scale ceph-nvmeof=1 ceph-nvmeof
Creating network "ceph-nvmeof_default" with the default driver
Creating volume "ceph-nvmeof_ceph-conf" with default driver
Creating ceph-cluster ... done

ERROR: for ceph-nvmeof  Container "12dc650f465a" is unhealthy.
ERROR: Encountered errors while bringing up the project.
make: *** [up] Error 1
➜  ceph-nvmeof git:(build-container) ✗ make logs
docker-compose logs --follow --tail=40
Attaching to ceph-cluster
ceph-cluster            | grep: CMakeCache.txt: No such file or directory
ceph-cluster            | ceph-mgr dashboard not built - disabling.
ceph-cluster            | rm -f core* 
ceph-cluster            | hostname 12dc650f465a
ceph-cluster            | ip 172.25.0.2
ceph-cluster            | port 10000
ceph-cluster            | ./vstart.sh: line 670: /etc/ceph/ceph.conf: Permission denied
ceph-cluster exited with code 1
➜  ceph-nvmeof git:(build-container) ✗ 
```

This is the suggested workaround, maybe though there is more elegant ways to address the issue.
See also the second part bellow, triggering volume ownership change.